packages/mds-ingest-service/repository/entities/event-entity.ts

```diff
@@ -51,6 +52,8 @@ export class EventEntity extends MigratedEntity(IdentityColumn(RecordedColumn(cl
   service_area_id: Nullable<UUID>
 
   telemetry?: TelemetryEntityModel
+
+  annotation?: EventAnnotationEntityModel
```

Comment:
Grr I really hate that we have to do this, is this considered best-practices with TypeORM for how to handle joins? Seems super weird to me that it's a part of a single entity model. Cc. @drtyh2o @nessur

Reply:
yes, and you can go one step further with annotations like `@OneToOne` or `@OneToMany`, if you want to start doing lazy-loading, relationship-loading, relationship-saving features with your models.

Reply:
I feel like perhaps we want to add those annotations to make it explicit that this is a Joined column, and not something that should ever actually be persisted in the DB then.

Reply:
If you add `@OneToOne` of other such decorators, then TypeORM may try to create some FK relationships, IIRC.